gallery/src/PageHeader/PageHeader.scss

```diff
@@ -0,0 +1,50 @@
+.nx-gallery-page-header {
+
+  &__inner {
```

Comment:
I feel like there's still a serious miscommunication here.  The new header should be taking advantage of the existing styles in RSC to the maximum extent that they are applicable, and should be doing so by using all relevant RSC classes (`.nx-page-header__inner` for instance). Then the gallery should add its own classes with their own style overrides as-needed.  We shouldn't be repeating styles that RSC's `.nx-page-header` and friends already have solved here.

Reply:
> I feel like there's still a serious miscommunication here. The new header should be taking advantage of the existing styles in RSC to the maximum extent that they are applicable, and should be doing so by using all relevant RSC classes (`.nx-page-header__inner` for instance). Then the gallery should add its own classes with their own style overrides as-needed. We shouldn't be repeating styles that RSC's `.nx-page-header` and friends already have solved here.

This was my bad, I updated it here: 188ca23b4a36433fcc202fd279284a0dafbf5dbf